Makefile

```diff
@@ -46,6 +46,7 @@ compile-windows:
 
 compile-linux:
 	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags "${LINKER_FLAGS} ${RELEASE_BUILD_LINKER_FLAGS}" -o ${DESTINATION}-amd64 ./cmd/copilot
```

Comment:
how much of a pain is it to update this one to `-linux-amd64`? 

I think the buildspec needs to be updated as well

Reply:
oky doky. i kept around the old one as well, until we can update our pipeline.

Reply:
This shouldn't require a pipeline update fortunately, since artifact releases are done by looping over the objects listed in the manifest file.